README.md

```diff
@@ -0,0 +1,104 @@
+# VSC Accounting - Brussel
+
+## Description
+
+Toolset to generate accurate accounting reports about the computational resources used in an HPC cluster.
+
+Main features:
+* retrieval of job log records from an ELK instance
+* retrieval of user account data from a local cache file and/or the VSC Account page
+* computation of usage statistics based on used compute time, executed jobs or unique active users
+* creation of accounting reports in various formats (see below)
+
+## Requirements
+
+Software dependencies
+* Python 3.6+
+  * vsc-base
+  * vsc-utils
+  * vsc-config
+  * vsc-accountpage-clients
+  * appdirs
+  * numpy
+  * pandas
+  * matplotlib
+  * beautifulsoup4
+  * elasticsearch
+  * elasticsearch_dsl
+
+Infrastructure
+* Access to ELK server with the logstash of your job scheduler
+* Access to VSC Account page (optional)
+
+## Example Use
+
+The following example generates 3 accounting reports (`-c`):
+* **compute-percent**: total use of compute time (in percentage)
+* **running-jobs**: total amount of running jobs
+* **top-users-percent**: ranking of users by percentage of compute time
+* **top-sites-percent**: ranking of research sites by percentage of compute time
+
+Reports will cover the first six months of 2020 (`-s 2020-01-01 -e 2020-06-30`), only consider two specific groups of nodes in the cluster (`-n node-group1 node-group2`) and will be made in HTML format (`-f html`):
+
+```bash
+$ accounting-report -s 2020-01-01 -e 2020-06-30 -f html -o general-nodes -n node-group1 node-group2 -c compute-percent running-jobs top-users-percent top-sites-percent
+```
+
+## Data Sources
+
+### ElasticSearch
+
+The retrieval of data requires an ElasticSearch instance containing the log records from the job scheduler. The accounting queries for `JOB_END` events and uses those records to calculate the use of compute resources with precision to the second.
+
+Supported resource managers:
+* Torque
+
+### User account data
+
+The main source of user account data is a local cache file in JSON format. Records in it will be added as users are found in the accounting data. Any account data can be manually added to the local cache file. By default it will be located in `~/.local/share/vsc-accounting/userdb-cache.json`.
```

Comment:
I prefer that it stays in `~/.local/share` because cache files in `.cache` can be removed at any time and, even though `userdb-cache.json` is indeed a cache for the VSC account page, it can also contain netid records not available elsewhere.